### DISCUSSION TITLE
Drag conflict with scroll on mobile device

### DISCUSSION THREAD

**Discussion by xufromxu** [pmndrs/use-gesture#607] [Q&A] [answered]
**description**
>I tried making a draggable pop-up that was draggable. When I drag,I expect the pop-up sheet follow the gesture, but it doesnt or just directly disappears. I made a replica based on the official example. https://codesandbox.io/s/heuristic-marco-27g94s?file=/src/App.jsx

I'm using this library for the first time, and do you guys have any ideas  to handle this situation. 
In order to reproduce this situation：
1. the content in popup sheet has overflow-scroll attr.
2. on mobile device emulation

**Comment by xufromxu** [comment, marked as answer]
add ` pointer: {
				touch: true,
			},` in config will work

**Reply by dbismut** [reply]
The answer is that you should use the css property touch-action: none as explained numerous times here and in the documentation.

**Reply by xufromxu** [reply]
I tried, but if I add touch-action:none, I cant scroll the content inside parent dom

**Reply by dbismut** [reply]
touch-action: pan-y then.
